<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Euclid::PhzQtUI::FormAuxDataManagement</class>
 <widget class="QWidget" name="Euclid::PhzQtUI::FormAuxDataManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>17</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Aux. Data Management :</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Import, organize or remove aux. data files to the local database</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_10">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tab_Management">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Filter Transmissions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <spacer name="horizontalSpacer_15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_FilterImport">
           <property name="text">
            <string>Import</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_FilterSubGroup">
           <property name="text">
            <string>Create (Sub)-Group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_FilterDelete">
           <property name="text">
            <string>Delete Selection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>The full transmission curves for the filter</string>
         </property>
         <property name="text">
          <string>List of Available Filter Curves  :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView_ManageFilter">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>SEDs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <spacer name="horizontalSpacer_16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_SedImport">
           <property name="text">
            <string>Import</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_SedSubGroup">
           <property name="text">
            <string>Create (Sub)-Group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_SedDelete">
           <property name="text">
            <string>Delete Selection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>List of Available SEDs :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView_ManageSed">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Reddening Curves</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <spacer name="horizontalSpacer_17">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_RedImport">
           <property name="text">
            <string>Import</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_RedSubGroup">
           <property name="text">
            <string>Create (Sub)-Group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_RedDelete">
           <property name="text">
            <string>Delete Selection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>List of Available Reddening Curves :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView_ManageRed">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Luminosity Function Curves</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <spacer name="horizontalSpacer_18">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_LuminosityImport">
           <property name="text">
            <string>Import</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_LuminositySubGroup">
           <property name="text">
            <string>Create (Sub)-Group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_LuminosityDelete">
           <property name="text">
            <string>Delete Selection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>The Luminosity Function (of the Magnitude or in Flux)</string>
         </property>
         <property name="text">
          <string>List of Available Luminosity Function Curves  :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView_ManageLuminosity">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
